src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -494,6 +494,9 @@ def __init__(self, logger=None, dbi=None, params=None):
         self.constraints["01_idx_wmbs_file_checksums"] = \
             """CREATE INDEX idx_wmbs_file_checksums_file ON wmbs_file_checksums(fileid) %s""" % tablespaceIndex
 
+        self.constraints["01_idx_wmbs_pnns"] = \
+            """CREATE INDEX idx_wmbs_pnns_pnn ON wmbs_pnns(pnn) %s""" % tablespaceIndex
```

Comment:
Apparently I forgot creating this index when this code was put in place, in the previous release.

Reply:
Nevermind, the primary key (and apparently unique) constraint already creates an index for that column. Removing it